### PR TITLE
calculator now has max length and max value

### PR DESCRIPTION
**Pull request by jboretsky**
calculator has max length of 17 (which is width of calculator) and max number of 999......

### CODE REVIEW

File: public/javascripts/moneyTracker.js [hunk 1]
```diff
@@ -65,7 +65,7 @@ app.run(function(editableOptions) {
                     $scope.addtoken=false;
                     $scope.subtracttoken=false;
                     $scope.eqtoken = false;
-                }else{
+                }else if($scope.output.length<17){
                     if(!(num == 0 && $scope.output == "-")){
                         $scope.output += String(num);
                     }
```
Comment (andreirtaylor) [inline public/javascripts/moneyTracker.js:71]: Wrong spacing

File: public/javascripts/moneyTracker.js [hunk 2]
```diff
@@ -65,7 +65,7 @@ app.run(function(editableOptions) {
                     $scope.addtoken=false;
                     $scope.subtracttoken=false;
                     $scope.eqtoken = false;
-                }else{
+                }else if($scope.output.length<17){
```
Comment (andreirtaylor) [inline public/javascripts/moneyTracker.js]: Why 17?

**Comment by jboretsky** [comment]
right, forgot that it'll be different for all devices, will make it shorter

**Comment by andreirtaylor** [comment]
Yeah I think 10 is a great number. Lets merge baby.

**Comment by adamtheschmidt** [comment]
The only game off the top of my head that might violate this is the millionaire version of monopoly.

we should consider going to scientific notation if it exceeds the width of the calculator

**Pull request merged**